docs/index.rst

```diff
@@ -178,7 +178,7 @@ out::
     pybabel init -l en_US -d . -i javascript.pot -D javascript
     perl -pi -e 's/fuzzy//' en_US/LC_MESSAGES/javascript.po
     pybabel compile -d . -D javascript
-    open http://0:8000/en-US/jsi18n/
```

Comment:
Yeah I think that makes sense - I like having a host name mapped e.g. addons.dev or WHY. The biggest nit is docker-machine and it's auto-ip assignment though.